pants-plugins/sample_conf/rules.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2021 The StackStorm Authors.
```

Comment:
Did you really write this file in 2021? Or should copyright be 2022 or 2023?

Reply:
Hmm. Yeah, that was copy pasta. Looks like I wrote this plugin in June of 2022. I've updated it to 2023 because that's when it will be merged.